crates/node/src/config.rs

```diff
@@ -19,6 +23,12 @@ pub struct EvolvePayloadBuilderConfig {
     /// Optional mint precompile admin address sourced from the chainspec.
     #[serde(default)]
     pub mint_admin: Option<Address>,
+    /// Optional activation height for base-fee redirect; defaults to 0 when sink set.
+    #[serde(default)]
+    pub base_fee_redirect_activation_height: Option<u64>,
+    /// Optional activation height for mint precompile; defaults to 0 when admin set.
+    #[serde(default)]
+    pub mint_precompile_activation_height: Option<u64>,
```

Comment:
nit: personally would move each activation block beside the `Option<Address>` for the corresponding features